doc/tutorials/setup_logical_partition_scenario_on_nuc7i7dnb.rst

```diff
@@ -0,0 +1,264 @@
+.. _setup_logical_partition_scenario_on_nuc7i7dnb:
+
+Guidelines of Setup Logical Partition Scenario on NUC7i7DNB
+############################################################
+
+ACRN hypervisor supports logical partition scenario, in which the User OS (such
+as Clear Linux) running in a pre-launched VM can bypass the ACRN hypervisor and
+directly access isolated PCI devices. The following guidelines provide step by
+step instructions on how to set up the ACRN hypervisor logical partition
+scenario on NUC7i7DNB boards running two pre-launched VMs. The logical partition
+scenario on NUC7i7DNB is shown in
+:numref:`logical_partition_scenario_on_nuc7i7dnb`.
+
+.. figure:: images/logical_partition_scenario_on_nuc7i7dnb.png
+   :align: center
+   :width: 400px
+   :name: logical_partition_scenario_on_nuc7i7dnb
+
+   The logical partition scenario on NUC7i7DNB
+
+Prerequisites
+*************
+
+* `NUC7i7DNB <https://ark.intel.com/content/www/us/en/ark/products/130394/intel-nuc-board-nuc7i7dnbe.html>`_
+* Connecting to the serial port, described in :ref:`getting-started-apl-nuc`
+* NVMe disk
+* SATA disk
+* Storage device with USB interface (such as USB Flash
+  or SATA disk connecting with USB3.0 SATA converter etc)
+* Disable "Intel Hyper Threading Technology" in the BIOS.
+
+  There are two reasons to disable "Intel Hyper Threading Technology":
+
+    * To mitigate L1TF.
+    * To avoid interference on the logical partition scenario between logical
+      cores.
```

Comment:
@leilu This explanation may be note, can you update it? thanks!

Reply:
fix at doc v3 version.